### PR TITLE
Add software best practices to this repository

### PR DESCRIPTION
**Pull request by bvreede**
This PR adds various files contributing to software best practices for this repository:
- A license (including a separate CC-BY license for 3rd party content in `/content`)
- A citation file with author info
- Documentation for the project in the Readme
- Nuxt technical information is moved to a contributing guide.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+message: "If you use this software, please cite it using these metadata."
+cff-version: "1.2.0"
+authors:
+  -
+    family-names: Kalverla
+    given-names: Peter
+    orcid: "https://orcid.org/0000-0002-5025-7862"
+    affiliation: "Netherlands eScience Center"
+  -
+    family-names: Alidoost
```
Comment (Peter9192) [inline CITATION.cff:20]: @SarahAlidoost just checking that you agree to being listed as contributor. I think you've mostly contributed feedback during demo's, right?

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    given-names: Fakhereh
+    orcid: "https://orcid.org/0000-0001-8407-6472"
+    affiliation: "Netherlands eScience Center"
+  -
+    family-names: Liu
+    given-names: Yang
+    orcid: "https://orcid.org/0000-0002-1966-8460"
+    affiliation: "Netherlands eScience Center"
+  -
+    family-names: Verhoeven
```
Comment (Peter9192) [inline CITATION.cff:30]: @sverhoeven just checking that you agree to being listed as contributor. I think you've mostly contributed feedback during demo's, right?

File: README.md [hunk 3]
```diff
@@ -1,69 +1,50 @@
...
+lacking in documentation.
+
+To overcome these limitations this data catalogue was developed. It is
+primarily an overview of all data outputs of the project. Hardly any actual
+data is hosted on the catalogue itself – instead, it redirects to the original
+sources and provides additional context and documentation.
+
+The data catalogue makes it possible to explore pre-generated maps and other
+visualizations through intuitive controls. The underlying data is made
+available via Zenodo.
```
Comment (Peter9192) [inline README.md:40]:
```suggestion
visualizations through intuitive controls. The underlying data will be made
available via Zenodo.
```

**Review by Peter9192 — APPROVED** [review]
Thanks a lot for taking care of this!

The updated README shows that you've understood what data is in here and why, but I think in this case it is more important to focus on "what" we offer through the data catalogue and not "why". So I left some suggestions to slightly restructure/rephrase it.

File: CITATION.cff (hunk 1 above)
Comment (SarahAlidoost) [reply CITATION.cff:20]: Thanks for asking, you can remove me from the list as I did not have any active contributions in this repo. :)

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    affiliation: "Netherlands eScience Center"
+  -
+    family-names: Vreede
+    given-names: Barbara
+    orcid: "https://orcid.org/0000-0002-5023-4601"
+    affiliation: "Netherlands eScience Center"
+identifiers:
+  - type: doi
+    value: '10.5281/zenodo.5905676'
+    description: The concept DOI of the work.
```
[the remark this answers is not in the document]
Comment (bvreede) [reply CITATION.cff]: Oops. Good catch.

**Review by geek-yang — APPROVED** [review]

**Pull request merged**